test/functional/tests/fault_injection/test_fault_injection_opencas_load.py

```diff
@@ -0,0 +1,152 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+from time import sleep
+
+import pytest
+
+from api.cas import casadm, casadm_parser, cli, cli_messages
+from api.cas.cache_config import CacheMode, CleaningPolicy
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_tools import fs_utils
+from test_tools.dd import Dd
+from test_tools.disk_utils import Filesystem
+from test_utils.size import Size, Unit
+
+mount_point = "/mnt/cas"
+test_file_path = f"{mount_point}/test_file"
+
+
+@pytest.mark.parametrize("cache_mode", [CacheMode.WB, CacheMode.WO])
```

Comment:
Maybe instead of WB and WO use sth like:
@pytest.mark.parametrize("cache_mode", [mode for mode in CacheMode if
                                        CacheModeTrait.LazyFlush & CacheMode.get_traits(mode)])

in case there will be some new cache mode supporting lazy flush in the future.
Also please update the description few lines below.